docker-compose.yml

```diff
@@ -1,21 +1,24 @@
 mongodb:
-  image: mongo:latest
-  container_name: mongodb
-  command: mongod --smallfiles
-  ports:
-   - "27017:27017"
-  volumes:
-   - ./mongo:/data/db:rw
-  volume_driver: local
+  image: bitnami/mongodb:latest
```

Comment:
Sorry for the late reply. I think it had to do with https://github.com/docker-library/mongo/issues/158
But that might now be fixed on the monmgodb side.